core/core-keybindings.el

```diff
@@ -46,50 +50,40 @@ used as the prefix command."
          (major-mode-prefix-emacs (concat dotspacemacs-major-mode-emacs-leader-key " " (substring prefix 1))))
     (unless long-name (setq long-name name))
     (let ((prefix-name (cons name long-name)))
-      (if (fboundp 'which-key-declare-prefixes-for-mode)
-          (progn
-            (which-key-declare-prefixes-for-mode mode
-              full-prefix-emacs prefix-name
-              full-prefix prefix-name)
-            (when (and is-major-mode-prefix dotspacemacs-major-mode-leader-key)
-              (which-key-declare-prefixes-for-mode mode major-mode-prefix prefix-name))
-            (when (and is-major-mode-prefix dotspacemacs-major-mode-emacs-leader-key)
-              (which-key-declare-prefixes-for-mode mode major-mode-prefix-emacs prefix-name)))
-        (define-prefix-command command)
-        (evil-leader/set-key-for-mode mode prefix command)))))
+      (which-key-declare-prefixes-for-mode mode
+        full-prefix-emacs prefix-name
+        full-prefix prefix-name)
+      (when (and is-major-mode-prefix dotspacemacs-major-mode-leader-key)
+        (which-key-declare-prefixes-for-mode mode major-mode-prefix prefix-name))
+      (when (and is-major-mode-prefix dotspacemacs-major-mode-emacs-leader-key)
+        (which-key-declare-prefixes-for-mode mode major-mode-prefix-emacs prefix-name)))))
+
+(defun spacemacs/set-keys (key def &rest bindings)
```

Comment:
Also we should find a more explicit name for this question, we should have mentioned the term leader to echo the dotfile variable `dotspacemacs-leader-key`, problem is I don't know a good one: `spacemacs/set-leader-keys` ?
`spacemacs/leader-set-keys` ?